### ISSUE TITLE
Juxtaphrenic peak

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6695]
**Preferred term label:**
Juxtaphrenic peak

**Synonyms**

**Definition (free text, please give PubMed ID)**
A juxtaphrenic peak is a small triangular opacity based at the apex of the dome of a hemidiaphragm, associated with upper
lobe volume loss of any cause (eg, postirradiation fibrosis or upper lobectomy). It is most readily appreciated on a frontal chest radiograph. The peak is caused by upward retraction of the inferior accessory fissure or an intrapulmonary septum associated with the pulmonary ligament.

PMID:18195376

Comment
See Figure 32 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
